desc/compute/_metric.py

```diff
@@ -1823,3 +1823,74 @@ def _gradtheta(params, transforms, profiles, data, **kwargs):
 def _gradzeta(params, transforms, profiles, data, **kwargs):
     data["|grad(zeta)|"] = jnp.sqrt(data["g^zz"])
     return data
+
+
+@register_compute_fun(
+    name="gbdrift",
+    label="\\mathrm{gradB-drift}",
```

Comment:
can you write out the math formula here? like `\\mathrm{gradB-drift} = 1/B^2 \mathbf{b} ..."`

Same for the other qtys, so that when they're plotted it has the actual formula as well as the gs2/gx name

Reply:
Added LaTex expression.